pdk.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'ffi', '~> 1.9.0'
   spec.add_runtime_dependency 'gettext-setup', '~> 0.24'
   spec.add_runtime_dependency 'hitimes', '1.3.0'
-  spec.add_runtime_dependency 'json-schema', '2.8.0'
+  spec.add_runtime_dependency 'json-schema', '~> 2.8.1'
```

Comment:
Technically speaking, `~> 2.8` might be more suitable (and improve my life as a packager)…  What do maintainers think about it?